spring-web-reactive/src/main/java/org/springframework/web/reactive/result/method/annotation/ServerHttpMessageReader.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2002-2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.web.reactive.result.method.annotation;
```

Comment:
Because this class is in the `spring-web-reactive` module, I'm afraid [SPR-14664](https://jira.spring.io/browse/SPR-14664) won't use this because `ResourceHttpMessageWriter` is in the `spring-web` module, `org.springframework.http.codec` package.